docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -410,3 +410,192 @@ The tests execution returns one of the following exit codes.
 |Test report isn't generated
 |`RED`
 |===
+
+== HTTP Configuration
+
+Here one could find description of general HTTP properties applied across all the HTTP interactions in VIVIDUS.
```

Comment:
`This section describes ...` maybe?

Reply:
I used the same format as in expression and vividus steps docs